test/hooks.ts

```diff
@@ -481,7 +481,7 @@ test('beforeRetry hook with parseJson and error.response.json()', async t => {
 			hooks: {
 				beforeRetry: [
 					async ({error, retryCount}) => {
-						t.true(error instanceof HTTPError);
+						t.true(isHTTPError(error));
```

Comment:
You can add this as an additional assertion, but I would prefer to keep the original.